packages/extension_google_sign_in_as_googleapis_auth/example/lib/main.dart

```diff
@@ -11,10 +11,7 @@ import 'package:extension_google_sign_in_as_googleapis_auth/extension_google_sig
 import 'package:googleapis/people/v1.dart';
 
 GoogleSignIn _googleSignIn = GoogleSignIn(
-  scopes: <String>[
-    'email',
```

Comment:
email doesn't seem to be needed...

Reply:
I thought both 'profile' and 'email' were required to sign in, but not seeing profile there is suspicious :P let me run this example real quick!

https://developers.google.com/identity/protocols/oauth2/scopes#google-sign-in

Reply:
('profile', 'email' and 'openid' seem to get added by the google sign-in API automatically if absent from the original request, good!)

Reply:
I've been running it all weekend – no issues!